.github/workflows/ci.yml

```diff
@@ -17,3 +21,8 @@ jobs:
 
       - name: Test
         run: npm run test
+
+      - name: Publish
```

Comment:
Do we really want to publish PRs or _only_ master?

Reply:
@christiansiegel I changed it to a new release job.